Sofa/Component/IO/Mesh/src/sofa/component/io/mesh/STLExporter.cpp

```diff
@@ -137,24 +137,24 @@ bool STLExporter::writeSTL(bool autonumbering)
     }
     if(!triangleIndices.empty())
     {
-        for(unsigned int i=0;i<triangleIndices.size();i++)
+        for(const auto& triangleIndex : triangleIndices)
         {
-            vecTri.push_back(triangleIndices[i]);
+            vecTri.push_back(triangleIndex);
         }
     }
     else if(!quadIndices.empty())
     {
         BaseMeshTopology::Triangle tri;
-        for(unsigned int i=0;i<quadIndices.size();i++)
+        for(const auto& quadIndice : quadIndices)
```

Comment:
```suggestion
        for(const auto& quadIndice : quadIndices)
```
and others in this file 🙊